src/energyplus/ForwardTranslator/ForwardTranslateAirLoopHVAC.cpp

```diff
@@ -155,6 +157,13 @@ namespace energyplus {
         }
       }
 
+      if (!fanOrUnitary) {
+        auto zoneHVACTerminalUnitVariableRefrigerantFlow = subsetCastVector<ZoneHVACTerminalUnitVariableRefrigerantFlow>(supplyComponents);
+        if (!zoneHVACTerminalUnitVariableRefrigerantFlow.empty()) {
+          fanOrUnitary = zoneHVACTerminalUnitVariableRefrigerantFlow.back();
+        }
+      }
+
```

Comment:
Similar to other "Unitary" systems... We'll write SPM mixed air only for nodes upstream of the VRF TU.